src/IssueCreator/IssueManager.cs

```diff
@@ -92,8 +93,8 @@ public async Task<bool> AddIssueToEpicAsync(long repoId, int epicNumber, long re
             {
                 //create an issue with just the number and repository set
                 // The only 2 parameters that need to be specified are the issueNumber and the repository.
-                Issue issue = new Issue(default, default,default, default, 
-                    number: issueNumber, default, default, default, default, default, default, default, default, default, default, default, default, default, default,default, default, default, 
+                Issue issue = new Issue(default, default, default, default,
```

Comment:
Whew! I noticed this but the PR was already merged and it wasn't a big deal, but kept triggering my anal-retentiveness. 🙂

Reply:
I used to have a reflection-based way of doing this... and when I removed it, I confused issue Id with issue number. *sigh*